cmd/maya-apiserver/cstor-operator/spc/runner.go

```diff
@@ -31,7 +31,12 @@ import (
 // is closed, at which point it will shutdown the workqueue and wait for
 // workers to finish processing their current work items.
 func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
-	defer runtime.HandleCrash()
+	var err error
+	defer func() {
```

Comment:
Why we are not doing fatal in caller of Run

Reply:
Run is called from Cobra if I'm not wrong.. so, added here